node_modules/@okta/qtip2/package.json

```diff
@@ -0,0 +1,105 @@
+{
+  "_args": [
+    [
+      "@okta/qtip2@3.0.4",
+      "/home/vagrant/okta/okta-signin-widget"
+    ]
+  ],
+  "_from": "@okta/qtip2@3.0.4",
+  "_id": "@okta/qtip2@3.0.4",
+  "_inCache": true,
+  "_location": "/@okta/qtip2",
+  "_phantomChildren": {},
+  "_requested": {
+    "name": "@okta/qtip2",
+    "raw": "@okta/qtip2@3.0.4",
+    "rawSpec": "3.0.4",
+    "scope": "@okta",
+    "spec": "3.0.4",
+    "type": "version"
+  },
+  "_requiredBy": [
+    "/"
+  ],
+  "_resolved": "https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta/@okta/qtip2/-/@okta/qtip2-3.0.4.tgz",
+  "_shasum": "34c0c795995eebbbb6e428f7f367ea030dec0229",
+  "_shrinkwrap": null,
+  "_spec": "@okta/qtip2@3.0.4",
+  "_where": "/home/vagrant/okta/okta-signin-widget",
+  "author": {
```

Comment:
I don't think this should be here. It seems to me that we copied it from the original, but we need to change it in the okta/qtip2 repo